src/Common/IconCache.cs

```diff
@@ -0,0 +1,202 @@
+﻿// Smuxi - Smart MUltipleXed Irc
+//
+// Copyright (c) 2015 Carlos Martín Nieto
+//
+// Full GPL License: <http://www.gnu.org/licenses/gpl.txt>
```

Comment:
Sure, this is what MD generated, I figured it was set in some bit of the project files.